documentation/src/main/asciidoc/infinispan_server_guide/infinispan_server_guide.adoc

```diff
@@ -113,6 +113,48 @@ In a managed domain the end user cannot directly launch a server process from th
 
 The Host Controller synthesizes the server's configuration by combining elements from the domain wide configuration (from +domain.xml+) and the host-specific configuration (from +host.xml+).
 
+==== Connecting remotely via JMX to server in Domain mode
+Sometimes you want to monitor Infinispan MBeans via JMX in Domain mode. Infinispan MBeans (like cache statistics etc.) are not exposed by the host controller, you
+have to connect directly to the server. To do that, you have to perform following steps in domain.xml:
+
+* add <remoting-connector> to jmx subsystem:
+[source,xml]
+----
+
+<subsystem xmlns="urn:jboss:domain:jmx:1.3">
+    ...
+    <remoting-connector use-management-endpoint="false"/>
+</subsystem>
+
+----
+
+* add <connector> to remoting subsystem:
+[source,xml]
+----
+
+<subsystem xmlns="urn:jboss:domain:remoting:3.0">
+    ...
+    <connector name="remoting-connector" socket-binding="remoting" security-realm="ApplicationRealm"/>
```

Comment:
Not sure how this is going to work with the existing `http-connector`. That one should probably be commented out (it can't be used anyway, because there's no Undertow listener).